main.js

```diff
@@ -0,0 +1,116 @@
+const button = document.getElementById('search-button');
+button.addEventListener('click', getCountryInfo);
+
+const searchBar = document.getElementById('search-bar');
+searchBar.addEventListener('keyup', setQuery)
+
+const container = document.getElementById("countries");
+
+function setQuery(e) {
+    search = e.target.value;
+    if (e.keyCode === 13) {
```

Comment:
`.keyCode` is deprecated. Gebruik daarentegen `.key`. Bron: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode